benchexec/tablegenerator/react-table/src/tests/__snapshots__/Summary.test.js.snap

```diff
@@ -8673,7 +8673,7 @@ exports[`Render Summary for multi-table-only-columns.table.html 1`] = `
         <div
          className="cell"
          dangerouslySetInnerHTML={"212"}
-         title="avg: 53, max: 212, median: 0, min: 0, stdev: 91.79869280115049655695465610"
+         title="avg: 53, max: 212, median: 0, min: 0, stdev: 91.7986928011505"
```

Comment:
As no number of significant digits was given the value was rounded to javascripts default precision of 15 digits. 
@PhilippWendler is this an issue?